dataland-frontend/src/components/forms/CreateEuTaxonomyForNonFinancials.vue

```diff
@@ -0,0 +1,343 @@
+<template>
+  <Card class="col-12 page-wrapper-card p-3">
+    <template #title
+      ><span data-test="pageWrapperTitle">
+        {{ editMode ? "Edit" : "Create" }} EU Taxonomy Dataset for a Non-Financial Company/Service</span
+      ></template
+    >
+    <template #content>
+      <div v-if="waitingForData" class="d-center-div text-center px-7 py-4">
+        <p class="font-medium text-xl">Loading Eu Taxonomy For Non Financials data...</p>
+        <em class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index: 20; color: #e67f3f" />
```

Comment:
We are using this spinner line all across dataland. I think this justifies aggregating it somewhere (or at least put the inline-styles in a css-class)

Reply:
This might be a seperate refactoring task.

Reply:
Story created.